lib/pdd/source.rb

```diff
@@ -203,15 +203,29 @@ def get_json(query)
       JSON.parse res.body
     end
 
-    def find_github_user(email)
-      base_uri = 'https://api.github.com/search/users'
-      query = base_uri + "?q=#{email}+in:email&perpage=1"
+    def find_github_user(info)
+      email, author = info.values_at(:email, :author)
+
+      # if email is not defined, changes have not been committed
+      return if email.nil?
+
+      base_uri = 'https://api.github.com/search/users?per_page=1'
+      query = base_uri + "&q=#{email}+in:email"
       json = get_json query
+
+      # find user by name instead since users can make github email private
+      unless json['total_count'].positive?
+        return if author.nil?
+
```

Comment:
@mbao01 it's better to avoid empty lines inside methods: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html

Reply:
I agree. It makes sense since a method should just do one thing.